cms/appresolver.py

```
@@ -38,7 +39,11 @@ def applications_page_check(request, current_page=None, path=None):
     for lang in get_language_list():
         if path.startswith(lang + "/"):
             path = path[len(lang + "/"):]
-    for resolver in APP_RESOLVERS:
+    try:
+        current_site = Site.objects.get_current(request)
+    except Site.DoesNotExist:
+        current_site = None
```

we should fail fast here.
I suggest `return None`